build/source/driver/multi_driver.f90

```diff
@@ -709,6 +741,12 @@ program multi_driver
 
   ! loop through HRUs
   do iHRU=1,gru_struc(iGRU)%hruCount
+   
```

Comment:
While not part of this PR, this basic parallelization highlights the need to differentiate between the different HRU indices. Here `iHRU` is the local HRU index (i.e., the index within a GRU) instead of the global HRU index used in the input and output files. Suggest change the name to `localHRU`